### ISSUE TITLE
a11y: convert divs and spans to buttons

### ISSUE THREAD

**Issue by NeonSpork** [Antlered-Viking/linktank#22]
All divs and spans that do button functionality should be converted to buttons to be able to be focused with keyboard and screen readers
